firebase-vscode/webviews/SidebarApp.tsx

```diff
@@ -160,6 +165,18 @@ export function SidebarApp() {
           setHostingInitState={setHostingInitState}
         />
       )}
+
+      {
+        // Only load quickstart panel if this isn't a Monospace workspace
+        !env?.isMonospace && (
```

Comment:
We really need to add a gating mechanism other than "isMonospace" for any new features that haven't been agreed to by the FireSQL team, but I can add it after this PR is merged.